libs/designer/src/lib/ui/templates/templatesStrings.ts

```diff
@@ -35,11 +35,6 @@ export const useTemplatesStrings = () => {
         id: '83Vrgj',
         description: 'Label for template',
       }),
-      DESCRIPTION: intl.formatMessage({
-        defaultMessage: 'Description',
-        id: 'FOoLuS',
-        description: 'Label for description',
-      }),
     },
```

Comment:
Please do not remove anything from here, you can combine two resource strings() whenever you are using... We are using this in other exported components